packages/ui-components/src/components/ContentHeading/ContentHeading.stories.tsx

```diff
@@ -20,14 +23,4 @@ const Template = (args: ContentHeadingProps) => <ContentHeading {...args} />
 
 export const Default = {
   render: Template,
-  parameters: {
-    docs: {
-      description: {
-        story: "The the main heading of the content area of a page/view.",
-      },
-    },
-  },
-  args: {
-    heading: "My Page Heading",
-  },
```

Comment:
In the preview there is no default text for the heading.
<img width="1076" alt="Screenshot 2024-11-29 at 09 56 26" src="https://github.com/user-attachments/assets/21281d4e-a2d9-4a19-a590-1bb0ccfef09a">

Reply:
Ooops! I should check after last minute quick changes. 🤪 Thank Youuu! Done.